### ISSUE TITLE
Provide a cli

### ISSUE THREAD

**Issue by lu-zero** [alexcrichton/rustc-demangle#24]
Would be nice to have a simple cli, possibly compatible with c++filt.

**Comment by alexcrichton** [comment]
I believe that's handled by https://crates.io/crates/rustfilt!

**Comment by lu-zero** [comment]
Oh, I didn't notice it, probably it will need an update once the new mangling rules land :)
